config-linux.md

```diff
@@ -267,9 +267,9 @@ The following parameters can be specified to setup the controller:
 
 ```json
     "memory": {
-        "limit": 0,
+        "limit": 536870912,
         "reservation": 0,
```

Comment:
now I have a question. @mrunalp or @crosbymichael, does this `reservation` correspond to `memory.soft_limit_in_bytes`? if so, that can take a string like `1G` or `256M`. So is this `uint64` value just the number of bytes for a soft limit?